### DISCUSSION TITLE
Cant convert to onnx yolov5s model with Focus layer

### DISCUSSION THREAD

**Discussion by kelectro** [zhiqwang/yolov5-rt-stack#456] [Q&A] [answered]
I am trying to convert to Onnx with Nms one custom model (yolov5s) but it seems that when I have the focus layer as first layer i get the following error :
```
line 1130, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'Conv' object has no attribute 'weight' 
```
I am using the code from [here](https://zhiqwang.com/yolov5-rt-stack/notebooks/export-onnx-inference-onnxruntime.html) to export onnx. If I repeat the process with a newer yolov5 model (yolov5s6)(without the Focus layer) it works fine. Is there any way that I can make the convertion with this model?

**Comment by zhiqwang** [comment, marked as answer]
Hi @kelectro , Which model you're using now?

You can specify the `version` parameter as follows if you're using v5.0 release.
```python
model = YOLOv5.load_from_yolov5(
    model_path,
    size=640,
    size_divisible=32,
    score_thresh=0.35,
    nms_thresh=0.45,
    version="r4.0",
)
```

**Reply by kelectro** [reply]
Yes, exactly!

**Reply by zhiqwang** [reply]
Then you can specify the `version` parameter as above.

**Reply by kelectro** [reply]
Ooh yes, I see. Thank you for the help!

**Reply by zhiqwang** [reply]
I marked this as an answer, please cancel this answer if you are still experiencing problems and feel free to open a new ticket if you encounter other questions.
